### ISSUE TITLE
Accessibility - `No records found` after the filtering the scheduler not announced by the Voice Over

### ISSUE THREAD

**Issue by ghulamghousdev** [bryntum/support#10215]
[Forum post](https://forum.bryntum.com/viewtopic.php?f=44&t=30732&p=155993#p155993)

Hi,

We have enabled the filter functionality for resources in the scheduler. When searching and no resources are found, we are using the Bryntum localized string to display "No technicians found" (shown in the attachment).

Our accessibility team has raised an issue where this text is not being announced by the narrator. Is there a way we can control this component or the role so that it is always announced?
![no-resources-found.png](https://forum.bryntum.com/download/file.php?id=31213)
